### DISCUSSION TITLE
Does anyone have the same idea as me🎃

### DISCUSSION THREAD

**Discussion by fromwhite** [pmndrs/drei#1641] [Ideas]
Is it possible for Text/Text3D to support emoji? 
Or Twitter emoji svg

**Comment by isaac-mason** [comment]
`<Text />` can support emojis if you provide an emoji font, e.g. [Noto Emoji](https://fonts.google.com/noto/specimen/Noto+Emoji). Here's an example: 

https://sketches.isaacmason.com/sketch/threejs-journey/13-text
https://github.com/isaac-mason/sketches/blob/main/src/sketches/threejs-journey/13-text/index.sketch.tsx

For `<Text3D />`, you should be able to create a json typeface for an emoji font. See the typeface json file docs in the readme - https://github.com/pmndrs/drei#text3d

**Reply by fromwhite** [reply]
Thank you.
